erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -67,6 +90,28 @@ def compute_error_tree(analyzer,
                        max_depth=DEFAULT_MAX_DEPTH,
                        num_leaves=DEFAULT_NUM_LEAVES,
                        min_child_samples=DEFAULT_MIN_CHILD_SAMPLES):
+    """Computes the error tree for the given dataset.
+
+    :param analyzer: The error analyzer containing the categorical
+        features and categories for the full dataset.
+    :type analyzer: BaseAnalyzer
+    :param features: The features to train the surrogate model on.
+    :type features: numpy.array or pandas.DataFrame
```

Comment:
Is this numpy.ndarray or numpy.array?

import numpy as np
np.ndarray
<class 'numpy.ndarray'>
np.array

Seems like numpy.array is built-in function array.

Reply:
done, replaced numpy.array with numpy.ndarray everywhere